torchgen/model.py

```diff
@@ -292,7 +292,9 @@ class DeviceCheckType(Enum):
     ExactSame = 1
 
 
-ViewSchemaKind = Enum("ViewSchemaKind", ("aliasing", "inplace", "out", "non_aliasing"))
+ViewSchemaKind = Enum(
+    "ViewSchemaKind", ("aliasing", "inplace", "out", "mutable", "non_aliasing")
+)
```

Comment:
I admit to being a bit confused here now, what's the difference between ViewSchemaKind and SchemaKind? Does ViewSchemaKind only apply to views? If so, why does it have "mutable"?

Reply:
If it's "view schema kind" (aliasing + non_aliasing) PLUS schema kind, we probably should have just done a Union of the two enums

Reply:
> If it's "view schema kind" (aliasing + non_aliasing) PLUS schema kind, we probably should have just done a Union of the two enums

yeah it's exactly that. The only place we actually use this enum is where we try to group up view ops into `NativeFunctionsViewGroup` objects. We basically want to say "if the op is a view op, treat it specially, otherwise fall back to the normal schema kinds".

There are probably a couple ways to slice it, but I'm thinking of removing the enum all together, and just have the grouping logic explicitly do that.